tinygrad/tensor.py

```diff
@@ -2333,6 +2333,19 @@ def sigmoid(self):
     ```
     """
     return F.Sigmoid.apply(self.cast(least_upper_float(self.dtype)))
+  def hardsigmoid(self, alpha:float=1/6, beta:float=0.5):
+    """
+    Applies the Hardsigmoid function element-wise.
+    NOTE: default `alpha` and `beta` values is taken from torch and differs from other common implementations
```

Comment:
i don't know what other common implementations are and find this odd, i think either say what the common one is, or just say it's the same as torch?